src/objective-c/tests/run_tests.sh

```diff
@@ -22,15 +22,29 @@ cd $(dirname $0)
 
 # Run the tests server.
 
-BINDIR=../../../bins/$CONFIG
+BAZEL=../../../tools/bazel
```

Comment:
I remember you said you're gonna remove this file

Reply:
Yeah will do.